From: Jiri Slaby <jslaby@suse.cz>
Date: Mon, 18 Nov 2019 10:17:06 +0100
Subject: syscalls: remove arch_prctl from x86_32
Patch-mainline: submitted 2019/11/18
References: build fix

It's not defined there:
/usr/lib/gcc/i586-suse-linux/9/../../../../i586-suse-linux/bin/ld: tables.o:(.data.rel+0x600): undefined reference to `syscall_arch_prctl'

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 include/syscalls-i386.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/syscalls-i386.h b/include/syscalls-i386.h
index 844ef68b27f9..1d365ed7367f 100644
--- a/include/syscalls-i386.h
+++ b/include/syscalls-i386.h
@@ -395,7 +395,6 @@ struct syscalltable syscalls_i386[] = {
 	{ .entry = &syscall_pkey_alloc },
 	{ .entry = &syscall_pkey_free },
 	{ .entry = &syscall_statx },
-	{ .entry = &syscall_arch_prctl },
 	{ .entry = &syscall_io_pgetevents },
 	{ .entry = &syscall_rseq },
 	{ .entry = &syscall_semget },
-- 
2.24.0

